RPIPs/RPIP-10.md

```diff
@@ -123,6 +127,20 @@ committee's membership is to be replaced (eg, due to term limits or a vote).
   - The selected membership SHALL be the nominees with the highest N vote weights, where N is the
     desired number of selections. In the event of a tie, the pDAO treasurer SHALL randomly select
     which of those nominees are selected.
+- Terms and Vacancies
+  - MCs SHALL be fully re-selected between 10-14 months from their last full selection.
+    - MCs SHOULD be re-selected approximately yearly.
+    - Selection timing MAY consider convenience (holiday timing, conventions, etc).
+    - Selection timing MAY consider efficiency (aligning multiple elections).
+  - If a vacancy occurs within six months of the previous selection, the next highest vote recipient
+    from the previous election SHALL be offered the seat, with the MC going down the list of
+    vote-receiving nominees until the position is filled.
```

Comment:
a timeframe should be specified in which offers must be responded to. I would suggest 24-48 hours.